src/components/List/List.component.jsx

```diff
@@ -37,6 +37,7 @@ const List = props => {
           </div>
         </>
       )}
+      <> </>
```

Comment:
I believe line 48 should be removed. See below code from commit 6f31581a7e385cfb9b59c2a3eb7ed32348e7448a

```javascript
import React, { useState, useEffect } from 'react';
import Listener from '../../services/Listener/Listener.service';

import { Footer } from '../component.index';
import { useHistory } from 'react-router-dom';

const List = props => {
  const listProps = props.location.state;
  const token = listProps.localToken;
  const [localToken, setLocalToken] = useState(token);

  const history = useHistory();
  // Redirect to the Add Item View
  const redirectAddItem = () => {
    history.push('/addItem', { localToken: localToken });
  };

  useEffect(() => {
    setLocalToken(token);
  }, []);

  return (
    <div className="list__page">
      <h1 className="page__title">List</h1>

      {localToken ? (
        <>
          <Listener localToken={localToken} />
          <br /> <br />
          <button onClick={redirectAddItem}> Add Item</button>
        </>
      ) : (
        <>
          <div>
            <p> Your shopping list is currently empty.</p>
            <button onClick={redirectAddItem}> Add Item</button>
          </div>
        </>
      )}
      <Footer />
    </div>
  );
};

export default List;

```

Reply:
It was temporary added so `Home` and `Add Item` would not get on top of each other. 
Would be removed once we apply the styling.